### PR TITLE
Implement fixed score setting

### PR DESCRIPTION
**Pull request by Hoikas**
In preparation for non-gamemgr CGZM games, this implements support for fixed score setting messages.

### CODE REVIEW

File: AuthServ/AuthDaemon.cpp [hunk 1]
```diff
@@ -896,6 +896,46 @@ void dm_auth_transferScorePoints(Auth_TransferScore* msg)
...
+    }
+    uint32_t scoreType = strtoul(PQgetvalue(result, 0, 0), 0, 10);
+    PQclear(result);
+    if (scoreType != Auth_UpdateScore::e_Fixed) {
+        SEND_REPLY(msg, DS::e_NetScoreWrongType);
+        return;
+    }
+    parms.set(1, msg->m_points);
+    result = PQexecParams(s_postgres,
+                          "UPDATE auth.\"Scores\" SET \"Points\"=$2 WHERE idx = $1",
```
Comment (zrax) [inline AuthServ/AuthDaemon.cpp]: Inconsistent spaces

File: AuthServ/AuthDaemon.cpp [hunk 2]
```diff
@@ -896,6 +896,46 @@ void dm_auth_transferScorePoints(Auth_TransferScore* msg)
...
+                                   "SELECT \"Type\" FROM auth.\"Scores\" WHERE idx=$1",
+                                    1, 0, parms.m_values, 0, 0, 0);
+    if (PQresultStatus(result) != PGRES_TUPLES_OK) {
+        fprintf(stderr, "%s:%d:\n    Postgres SELECT error: %s\n",
+                __FILE__, __LINE__, PQerrorMessage(s_postgres));
+        PQclear(result);
+        SEND_REPLY(msg, DS::e_NetInternalError);
+        return;
+    }
+    if (PQntuples(result) != 1) {
```
Comment (zrax) [inline AuthServ/AuthDaemon.cpp]: How does a new `e_Fixed` score entry get created?

Comment (Hoikas) [inline AuthServ/AuthDaemon.cpp]: When the client requests the creation of one in `dm_auth_createScore`

File: AuthServ/AuthDaemon.cpp (hunk 1 above)
Comment (Hoikas) [inline AuthServ/AuthDaemon.cpp]: fixed

File: AuthServ/AuthDaemon.cpp (hunk 2 above)
Comment (zrax) [inline AuthServ/AuthDaemon.cpp]: Ah, of course >.<

**Pull request merged**